File: 1737-maximum-nesting-depth-of-the-parentheses/maximum-nesting-depth-of-the-parentheses.c
int maxDepth(char* s) {
    int count = 0, max = 0;
    for (int i = 0; i < strlen(s); i++) {
        if (s[i] == '(') {
            count++;
            if (count > max)
                max = count;
        } else if (s[i] == ')') {
            count--;
        }
    }
    return max;
}

File: 1737-maximum-nesting-depth-of-the-parentheses/maximum-nesting-depth-of-the-parentheses.py
class Solution:
    def maxDepth(self, s: str) -> int:
        depth = 0
        maximum = 0
        for char in s:
            if char == "(":
                depth += 1
                maximum = max(depth, maximum)
            elif char == ")":
                depth -= 1
        
        return maximum